examples/specs/window_running_average.vl.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A line graph showing how the average rating for a movie has changed over the years.",
+  "data": {"url": "data/movies.json"},
+  "width": 500,
+  "transform": [
+    {
+      "window": [{
```

Comment:
1) Don't you have to sort the data before hand.  
2) Shouldn't you calculate year first so you group the data by year, not by individual time in seconds.